.github/workflows/build.yml

```diff
@@ -89,24 +96,56 @@ jobs:
           fi
           echo "insecure_skip_tls_verify=true" >> $GITHUB_OUTPUT
 
+
+  chart-verifier:
+    name: Run chart-verifier
+    runs-on: ubuntu-22.04
+    needs: [setup]
+
+    outputs:
+      report_content: ${{ steps.check_report.outputs.report_content }}
+      redhat_to_community: ${{ steps.check_report.outputs.redhat_to_community }}
+      message_file: ${{ steps.pr_comment.outputs.message-file }}
+      webCatalogOnly: ${{ steps.check_pr_content.outputs.webCatalogOnly }}
+      chartEntryName: ${{ steps.check_pr_content.outputs.chart-entry-name }}
+      chartNameWithVersion: ${{ steps.check_pr_content.outputs.chart-name-with-version }}
```

Comment:
Normalize output keys here too. Yes, I know I used camel case in my testing, but I don't care if we go hyphens or camel case.

Reply:
We can do this in a separate PR so since it entails updating the python scripts. I will put that on my plate